.github/workflows/testing-scheduled.yml

```
@@ -5,16 +5,14 @@ on:
     - cron: '30 5 * * 1' 
 
 
-jobs:     
-  test-stable-scheduled:
-    uses: 
-      ./.github/workflows/testing.yml
-    with:
-      xdist: no
-      branch_name: stable
-      event_name: ${{ github.event_name }}
-    secrets:
-      PAT: ${{ secrets.PAT }}
+jobs:
+  trigger_stable_test:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-latest
    permissions:
      actions: write
```

While testing I found out that this permission is required to be able to use the runner token to trigger another workflow.